### ISSUE TITLE
[PROPOSAL 💡] - Do halvening on the wihdrawal amount percentage as well.

### ISSUE THREAD

**Issue by aayusharyan** [sunflower-land/sunflower-land#859]
Recently, another halvening was done which has now again reduced the earning potential of a farmer. Now with that hard earned money cannot be withdrawn easily because the barrier for exit is still as high as it was before halvening. This leads to increasing difficulty in getting the token out because say for less than 10SFL, the withdrawal fees is 30%. Now after halvening, it will take twice the amount of time to be able to earn the money and since fees is still unchanged, technically now a farmer will be spending twice the amount of game hours in terms of fees.

A proposal would be to reduce this threshold with halvening so the movement can be smooth for exchange of the tokens between players.

**Comment by Pranav-yadav** [comment]
Definitely.. New players point of view as well their gameplay must be taken into account... 
The Withdrawal fees (tax) *was* going to be updated *(decreased gradually)* over time... 

I think @adamhannigan  forgot to do that...

**Comment by akshay-kh** [comment]
Yeah this is a very good proposal hope core team consider it. Withdraw fee should also reduce in each halving. 

@aayusharyan thanks for creating this issue.

**Comment by adamhannigan** [comment]
Thanks for the contribution and feedback! ❤️ 

 We are moving towards using an ideas portal to help capture and vote on community ideas. Github will only be used for bug/technical reports. We are only a small team and it is becoming difficult for us to respond individually to every case so we have began using community voting software.   

 If you could please upload your idea to the following community portal it would be greatly appreciated 🤗   

 https://sunflower-land.ideas.aha.io/

 You can vote on similar ideas and help bring ideas to our attention. Thanks again for being apart of this community!
